src/test/mediapipe/python/scripts/bad_execute_wrong_signature.py

```diff
@@ -0,0 +1,26 @@
+#*****************************************************************************
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#*****************************************************************************
+
+class OvmsPythonModel:
+
+    def initialize(self, kwargs: dict):
+        return True
+
+    def execute(self, inputs: list, additional_argument, another_additional_argument) -> list:
```

Comment:
"bad execute wrong signature" but in fact it is wrong return type (None) - right?